roles/openshift_node/defaults/main.yml

```diff
@@ -195,6 +195,9 @@ oreg_auth_credentials_path: "{{ openshift_node_data_dir }}/.docker"
 oreg_auth_credentials_replace: False
 l_bind_docker_reg_auth: False
 openshift_use_crio: False
+l_crio_openshift_version: "{{ openshift_image_tag | default(openshift_release) | default('latest') | string }}"
+l_crio_use_new_var_sock: "{{  (l_crio_openshift_version == 'latest') or (l_crio_openshift_version | version_compare('3.9', '>='))  | bool }}"
```

Comment:
Unsure how best to accomplish the needed part in GCP CI check.

Just realized we can check openshift_version here.  That would be the mots appropriate thing in this case in this role.

```yaml
l_crio_use_new_var_sock: "{{ openshift_version | version_compare('3.9', '>=') }}"
```

Reply:
any risk that we don't do exactly the same thing as we do with node and the two configuration files get different values?

Reply:
Not really.  openshift_version is derived from openshift_release and/or openshift_image_tag and/or openshift_pkg_version.  Since the nature of this check is to determine if we're on 3.9 openshift or higher, I think we'll be okay.  If we start changing this type of stuff between point releases (eg, 3.9.0, 3.9.1) it's going to be painful.